### ISSUE TITLE
The `input` event is not fired on `paste` if the element is empty or is completely selected beforehand

### ISSUE THREAD

**Issue by VoimiX** [autoNumeric/autoNumeric#521]
After paste **addEventListener('input', (e))** event is not fired
It is fired only after loosing a focus 
browsers: ie, ff, chrome

live example
https://codepen.io/AnotherLinuxUser/pen/BweJOv?editors=1011

**Comment by AlexandreBonneau** [comment]
This is weird since the `input` event should correctly be sent (see the [code here](https://github.com/autoNumeric/autoNumeric/blob/742b80dcbeea6bd463df9444837aa944ac04c2d7/src/AutoNumeric.js#L6867)).

Also, we need to add tests for checking this.

**Comment by AlexandreBonneau** [comment]
Well, the AutoNumeric code is correct and does send the native `input` event on `paste`.

You can check that out on the [reproducible example here](https://codepen.io/AnotherLinuxUser/pen/XzreNJ?editors=1011).

Tip: When reporting a bug, it's better to re-create the bug yourself in a live example instead of using an already existing snippet!

Bonus: We already have tests in place to make sure this works, so, that's good! 😎

**Comment by VoimiX** [comment]
try to run this snippet 
https://codepen.io/voimix/pen/mqbKNG?editors=1011 (I commented only 2 options)
and paste any number

also changing value via wheel doesnt fire this 'input' event

**Comment by AlexandreBonneau** [comment]
Ok, the *very first* paste does not trigger an `input` event, while all the subsequent ones do.

About the changes happening after using the `wheel` event, could you please create a separate issue? Thanks for reporting that!
